theatre/studio/src/panels/SequenceEditorPanel/RightOverlay/FocusRangeZone/FocusRangeZone.tsx

```diff
@@ -107,6 +108,14 @@ function usePanelDragZoneGestureHandlers(
   layoutP: Pointer<SequenceEditorPanelLayout>,
   panelStuffRef: React.MutableRefObject<ReturnType<typeof usePanel>>,
 ) {
+  const [mode, setMode] = useState<'none' | 'creating' | 'moving-panel'>('none')
+
+  useCssCursorLock(
```

Comment:
`useCssCursorLock()` is more flexible than using `useDrag()`'s `lockCursorTo:` prop. You can change the cursor mid-flight for example.